cypress/integration/Stepper._spec.js

```diff
@@ -0,0 +1,227 @@
+        })
+
+        describe('Checks whther Back Button is working or not', () => {
+
+            it('Checks whether a user can convert CC BY-ND License into CC BY-SA', () => {
+                cy.get('.previous-button').click()
+                cy.get('.step-3')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-4')
+                .find('[value="no"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.recommended-card').should('be.visible').contains('CC BY-SA 4.0')
+                cy.get('.license-use-card').should('be.visible').contains('CC BY-SA 4.0')
+            })
+
+            it('Checks whether a user can convert CCO using the License Dropdown into CC BY-NC-SA with the answer to the first question as "NO" ', () => {
+                cy.visit('/')
+                cy.get('.step-0')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('select').select('CC0 1.0')
+                cy.get('.recommended-card').should('be.visible').contains('CC0 1.0 Universal')
+                cy.get('.next-button').click()
+                cy.get('.v-checkbox:nth-child(1) > input').check()
+                cy.get('.v-checkbox:nth-child(2) > input').check()
+                cy.get('.next-button').click()
+                cy.get('.license-use-card').should('be.visible').contains(' CC0 1.0 ')
+                cy.get('.previous-button').click()
+                cy.get('.previous-button').click()
+                cy.get('.previous-button').click()
+                cy.get('.step-0')
+                .find('[value="no"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-1')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-2')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-3')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-4')
+                .find('[value="no"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.recommended-card').should('be.visible').contains('CC BY-SA 4.0')
+                cy.get('.license-use-card').should('be.visible').contains('CC BY-SA 4.0')
+            })
+        })
+
+        describe('Checks whether the user can click on Start Again Button and application gets a hot reload', () => {
+            it('Clicks on Start Again Buttton', () => {
+                cy.get('.restart-button').click()
+                cy.get('.step-1').should('have.class','inactive')
```

Comment:
Please, add the check for `.recommended-card` and `.license-use-card`. They shouldn't be visible.